client/src/components/backend/Dashboard/Activity.js

```diff
@@ -5,35 +5,48 @@ export default class Activity extends Component {
 
   static displayName = "Dashboard.Activity";
 
-  static propTypes = {};
+  static propTypes = {
+    statistics: PropTypes.object
+  };
+
+  formatReaderIncrease(stats) {
+    const increase = stats.readerIncrease;
+    let out = "";
+
+    if (increase > 0) {
+      out = "+ " + increase.toString();
+    } else if (increase < 0) {
+      out = "- " + Math.abs(increase).toString();
+    } else {
+      out = "0";
+    }
+    return out;
+  }
```

Comment:
Simplify your code. Lines 12 through 23 can be reduced to 4 lines of flat code rather than 11 lines of nested code:

```
const increase = stats.readerIncrease;
if (increase > 0) return "+ " + increase.toString();
if (increase < 0) return "- " + Math.abs(increase).toString();
return 0;
```